packages/zk-certificates/scripts/mintBatchSBT.ts

```diff
@@ -0,0 +1,58 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import csv from 'csvtojson';
+import { ethers } from 'hardhat';
+
+/**
+ * Performs some batch mint of SBTs.
+ */
+async function main() {
+  // parameters for test interaction
+  const [owner] = await ethers.getSigners();
+  const SBTAddress = '0xA10F4f2e6e881B876dF5cAfEF754F42d81606656';
+
+  const dataPath = './data/mockSBT.csv';
+  let data;
+
+  await csv({ delimiter: ';' })
+    .fromFile(dataPath)
+    .then((jsonObj) => {
+      data = jsonObj;
+    });
+
+  console.log('operating owner:', owner.address);
+
+  const SBTInstance = await ethers.getContractAt(
+    'GalacticaTwitterSBT',
+    SBTAddress,
+  );
+
+  for (const user of data) {
+    const userAddress = user['wallet addresses'];
+    const name = user['SBT name'];
+    const symbol = user['SBT name'];
+    const uri = user['SBT description'];
+
+    console.log(`Giving ${userAddress} an NFT`);
+    const tx = await SBTInstance['mint(address,string,string,string)'](
```

Comment:
Shouldn't a script with the name mintBatch, call the mintBatch method?
It would be more gas efficient and require less txs, but it is some effort to adjust the mintBatch function in the script according to the new mint parameters.

Reply:
I agree, I will use this mintBatch.